% Template for PLoS
%subsubsection is 3rd level heading, PLOS doen't support 4th level.
% Version 3.3 June 2016
% Please follow the template instructions whenever possible.
%
% Once your paper is accepted REMOVE ALL TRACKED CHANGES.
% Use latexdiff to track changes during review to maintain a clean file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info
%
% There are no restrictions on package use within the LaTeX files except that
% no packages listed in the template may be deleted.
% Do not include colors or graphics in the text.
%
% LaTeX source should be a single file: don't use \input, \externaldocument
%
% -- FIGURES AND TABLES
%
% Include table/figure captions directly after paragraph where first cited.
%
% NO GRAPHICS
% - Figures should be uploaded separately from your manuscript file. 
% - Extract figures (in PDF) generated by LaTeX before submission.
% - Combine multiple panels/subfigures into one image before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column. 
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2". 
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % % 
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8]{inputenc}
%\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Leave date blank
\date{}

% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
\setlength{\headheight}{27.023pt}
\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
%\rfoot{\thepage/\pageref{lastpage}}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\sf PLOS}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION


\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Mutation Rate as a Selection Mechanism in Continuous Evolution of Bacteriophage} % Title Case
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Peter Reintjes\textsuperscript{1}
\\
\bigskip
\textbf{1} Museum of Life and Science, Durham, North Carolina, USA
\bigskip

% Additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
% 
% Remove or comment out the author notes below if they aren't used.
%
% Primary Equal Contribution Note
%\Yinyang These authors contributed equally to this work.

% Additional Equal Contribution Note
% Also use this double-dagger symbol for special authorship notes, such as senior authorship.
%\ddag These authors also contributed equally to this work.

% Current address notes
%\textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% \textcurrency b Insert second current address 
% \textcurrency c Insert third current address

% Deceased author note
%\dag Deceased

% Group/Consortium Author Note
%\textpilcrow Membership list can be found in the Acknowledgments section.

% Use the asterisk to denote corresponding authorship and provide email address in note below.
%* correspondingauthor@institute.edu

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Engineering novel proteins via continuous evolution of bacteriophage currently requires the host bacteria to be transformed with two additional functions: A mutagenesis vector to provide an elevated rate of viral mutation, and a selection mechanism to increase the number of infectious progeny for the individual encoding the improved product\cite{pace}.  I propose using a mutagenesis suppressor as the selection mechanism whereby the desired activity results in a higher number of clones, thus rewarding the genotype with more faithful progeny.

% Please keep the Author Summary between 150 and 200 words
% Use first person. PLOS ONE authors please skip this step. 
% Author Summary not valid for PLOS ONE submissions.   
%\section*{Author Summary}
%Lorem ipsum dolor sit amet,

\linenumbers

% Use "Eq" instead of "Equation" for equation citations.
\section*{Introduction}
Continuous evolution of bacteriophage has the potential to become a potent protein engineering tool\cite{pace}\cite{monsanto}.
For example, PACE - Phage Assisted Continuous Evolution described by Esvelt,Carlson and Lui. 
%M13 genes
%I(assembly),
%II(replication init),
%X (DNA accumulation, inside gII) 
%III(fusion),
%IV(assembly),
%V(destabilizing protein),
%VI(minor coat),
%VII(minor coat),
%VIII(major coat),
%IX(minor coat),
Continuous evolution rapidly produces a viral genome containing a gene
which has undergone many generations of mutation and selection for
a particular property.  The generality of this approach to protein engineering
is limited by our ability to insert an expressable initial gene into the phage and create a selection mechanism for the desired activity.

Specifically, the PACE system requires:
\begin{itemize}

\item{}
A modified viral genome replacing a crucial phage gene with a gene to be evolved.

\item{}
A transformed host with inducible mutagenesis.

\item{}
A host plasmid containing a selection mechanism to provide the crucial phage gene in proportion to the desired activity of the evolving gene.
\end{itemize}

\section{M-Selection}
Extremely high, yet controllable, in-vivo mutation rates are now possible\cite{mutation}.  This mutagenesis can be generated and controlled within an individual host cell.  We suggest that if this mutagenesis is initiated  by infection, say with the phage shock promotor but then reduced by the desired activity, the two principal components of evolution will be satisfied and the desired activity will be selected for as the number of phage progeny of the individual exhibiting that activity will be amplfied by proportionally more faithful reproduction.  We propose calling this mechanism Mutagenesis-selection or M-selection.


his technology maked With M-selection, these high rates will correspond to little or none of the desired activity, these lines of descent will be subject to catastrophic levels of mutation and low probabilities of faithful reproduction. The modulation of mutation rate within each host cell means that one of these wildly mutating strains could find an environment for faithful reproduction if it should stumble upon a promising mutation in the evolving gene.

The percentage of phage progeny which are exact copies of the parent is given by the Poisson distribution where $$\lambda$$ is the mutation rate per virion (mutation rate/base * 6000bp/ genome). The probability of a virion with zero mutations is:
\vskip 0.2cm
\begin{eqnarray}
\label{eq:POC}
\mathrm{P}\left( 0 \right) = \frac{{e^{ - \lambda } \lambda ^0 }}{{0!}}
\end{eqnarray}

Taking a phage production average of 100/hour for normal M13 infection requires us to have a clone rate of at least 2\%: Two faithful copies per infection in order to avoid washout.  A slower flow rate would lower this minimum fraction.

M-selection may allow for, or even require a flow rate which changes over the course of the experiment.  A ramping up of the flow rate would be appropriate

PACE is designed to keep the transit time of host cells through the lagoon to be on the order of one generation.  However, at very high levels of mutation it seems unlikely that multiple generations of viable E. coli mutants can be produced.

If the desired activity of the evolving protein produces a mutagenesis repressor, improvements in the desired activity will result in a higher percentage of faithful copies of the parent genome in the succeeding generation.

As the evolving protein approaches a high level of desired activity, the proportionally lowered mutation rate will ensure that more viral progeny contain exact copies of the parent genotype while a baseline mutation rate ensures that our mechanism to lower the mutation rate will not result in stagnation.

In PACE, the number of infectious progeny is proportional to the activity
of the selection mechanism. In principle, if the current mutation of the gene
produces a product with none of the desired activity, none of the infectivity
protein will be produced by the host, and there will be no infectous progeny.
This process involves a continuous flow of host cells through a reaction vessel
and so any genotype producing less than two infectious virions per host will
be washed out.

The benefits of M-Selection may include:

The virus is not crippled, that is, no critical genes need to be removed from the virion, and no phage genes are required in a host plasmid.  This avoids the problem associated with the presence of gIII in the host and improves our ability to propagate phage prior to the evolution experiment.  Of course, care must be taken to ensure that the evolving gene is not discarded before the experiment begins.  However, M13 with one additional gene(domain) is exactly what has been perfected in phage display.

The nominal (high) mutation rate can be chosen to guarantee that there will be
a minimum number of non-mutant copies produced for every infection.  This percentage of clones will then increase as the mutation rate is reduced.

As far as we know, modulation of the mutation rate within a particular host has
not been exploited in this way.  We refer to this as M-selection, wherein the selection mechanism consists simply of reducing the rate of mutation.

Figure 1 shows how the combined mutagenic and selection plasmid takes the place of the AP and MP in PACE.

\def\boxone#1{
  \put(19,30){\line(1,0){67}}     % horiz top
  \put(19,0){\line(1,0){67}}      % horiz bottom
  \put(20,0){\line(0,1){30}}      % vert left
  \put(85,0){\line(0,1){30}}      % vert right
  \put(25,15){#1}
}
\def\boxtwo#1#2{
  \put(19,30){\line(1,0){80}}     % horiz top
  \put(19,0){\line(1,0){80}}      % horiz bottom
  \put(20,0){\line(0,1){30}}      % vert left
  \put(98,0){\line(0,1){30}}      % vert right
  \put(25,17){#1}
  \put(25,7){#2}
}
\def\circleone#1#2{\circle{#1}\put(-17,-4){#2}}
\def\circletwo#1#2#3{\circle{#1}\put(-17,0){#2}\put(-15,20){#3}}

\picture(400,150)(-90,0)

\put(90,90){\vector(0,-1){15}}
\put(90,45){\vector(0,-1){14}}

\linethickness{2pt}
\thicklines
\put(0,15){\circleone{40}{\Large{\ \ P1}}}
\put(20,15){\vector(1,0){30}}

\put(35,90){\boxtwo{\ Phage Shock}{\ \ \ Promoter}}
\put(-20,55){
\fbox{%
    \begin{minipage}{1.0in}
Activity Induced\linebreak
\hspace*{5 mm}Repressor
    \end{minipage}%
}}
\put(64,60){\vector(1,0){18}}
\put(83,57){\fbox{\begin{minipage}{0.05in}
		\hfill\vspace{0.30in}
            \end{minipage}
           }}
\put(35,0){\boxtwo{MUTAGENESIS\ }{\ \ \ Components}}

\put(130,15){\vector(1,0){30}}
\put(180,15){\circleone{40}{\Large{\ \ F1}}}

\endpicture
\vskip 1.0cm
\centerline{\bf Fig 1. M-selection Plasmid}
\vskip 0.2cm

\section{Comparison with Natural Selection}
Selection pressure for an optimal mutation rate has its own natural history\cite{evomut} although for settled phyla we see a certain natural mutation rates.  Bacteriophage and multicellular eukaryote mutation rates are higher than bacteria and single cell eukaryotes (around 0.003).

A selection mechanism for lowered mutation rate would not seem to correspond to anything in the natural world.  Natural selection is largely about responses to the external environment, which we control closely in laboratory evolution.  Perhaps the closest parallel is found in species which can reproduce by asexual or sexual reproduction and change the rates between the alternatives as a way of accessing more diversity through sexual reproduction or producing more clones via asexual reproduction.


\section*{Materials and Methods}

\subsection*{Population Dyamics of Bacteriophage}
\subsubsection*{Clones versus Mutants}
Although it may sounds like the title of a recent movie, this precisely states the selection mechanism of of mutation.

Percentage of activated genes due to Phage Shock promoter.
Number of polIII-  (mutagenic polymerase) transcripts times error rate due to mutagenic polymerase.

Error rate of normal polII, polIII

Number of phage particles produced np(0-6 min) = 0 (six minutes before any phage appear),
then production between 75-200 per hour.

Initial single-strand (virion) DNA production occurs in non-mutagenic environment, then mutants appear along with polII- production, until suppression of polII- occurs due to binding.  Binding ? anti-sense RNA transcription ? polII-  repression ? reduction in mutations.

% Results and Discussion can be combined.
\section*{Results}

\subsection*{Comparison with Sexual/Asexual Reproduction}
Sexual reproduction is not the same as an elevated mutation rate, but it shares the characteristic that even a perfectly fit individual will not be producing faithful copied of its genome. For organisms with access to both forms, the percentage of offspring produced by asexual reproduction would be comparable to our percentage of true copies.
\subsubsection*{Yeast}
Although yeast has access to both sexual and asexual reproduction, the (time) cost of sexual reproduction is high and it is generally invoked under conditions of stress, where an increase in genetic variety of offspring.
\subsubsection*{Daphnia} 
In Daphnia, longevity and healthy survival to the end of the season result in so called 'winter eggs' which reproduce asexually, providing a percentage of offspring which are clones of the parent. The cost differential between sexual and asexual reproduction is low and each generation represents a mix of both.

\begin{enumerate}
	\item{react}
	\item{diffuse free particles}
	\item{increment time by dt and go to 1}
\end{enumerate}

\subsection*{Using RNA Antisense.}

The components for the broad spectrum, high level mutagenisis described in Badran and Liu\cite{mutation} are variations (often single mutation variants) of the functional domains of high fidelity DNA replication and as such are not suitable targets for RNA anti-sense interference. Any such anti-sense interference would be likely to affect their normal counterparts which are needed for faithful reproduction.

RNA anti-sense of gIV could be utilized to block the primary phage shock promotor which was the primary trigger activating the high level of mutation in the lagoon.

Lowering the mutation rate can be achieved in a number of ways.

\begin{itemize}
	\item Supress pIV production or otherwise block phage shock promotor
	\item Selectively block downstream components of mutagenesis
	\item Enhance production of wild-type high-fidelity replication components.
\end{itemize}

\section*{Discussion}
Modeling needs work.

\section*{Conclusion}

I think this will work.

\section*{Supporting Information}

% Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
\paragraph*{S1 Fig.}
\label{S1_Fig}
{\bf Bold the title sentence.} Add descriptive text after the title of the item (optional).

\paragraph*{S1 Appendix.}
\label{S1_Appendix}
{\bf Algorithm.} The code goes here.

\paragraph*{S1 Table.}
\label{S1_Table}
{\bf Graph maybe.} The Graph goes here.

\section*{Acknowledgments}
So many people to thank.

\nolinenumbers

% Alternative to directly entering bibtex entries:
% 1) Compile BiBTeX database using plos2015.bst style file
% 2) Paste the contents of your .bbl file here.
\begin{thebibliography}{10}

\bibitem{pace}
Esvelt K. M., Carlson J. C. \& Liu D. R.
\newblock {A system for the continuous directed evolution of biomolecules}.
\newblock Nature 472, 499–503 (2011) .

\bibitem{mutation}
Badran AH, Liu DR.
\newblock {{D}evelopment of potent in vivo mutagenesis plasmids with broad mutational spectra}.
\newblock Nature Communications. 2015;6:8425. doi:10.1038/ncomms9425.

\bibitem{evomut}
Lynch M.
\newblock {{E}volution of the Mutation Rate}.
\newblock Trends in Genetics, 2010 August ; 26(8): 345–352. doi:10.1016/j.tig.2010.05.003.

\bibitem{monsanto}
  Badran A. H., Guzov V.M., Huai Q., Kemp M. M.,Vishwanath P.,Kain W., Evdokimov A., Moshiri F., Turner K.H.P., Malvar T., Liu D.R.
  \newblock{{C}ontinuous evolution of Bacillus thuringiensis toxins overcomes insect resistance}
  \newblock  Nature 533,58-63 (2016) doi:10.1038/nature17938
\end{thebibliography}



\end{document}



