% Template for PLoS
%subsubsection is 3rd level heading, PLOS doen't support 4th level.
% Version 3.3 June 2016
% Please follow the template instructions whenever possible.
%
% Once your paper is accepted REMOVE ALL TRACKED CHANGES.
% Use latexdiff to track changes during review to maintain a clean file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info
%
% There are no restrictions on package use within the LaTeX files except that
% no packages listed in the template may be deleted.
% Do not include colors or graphics in the text.
%
% LaTeX source should be a single file: don't use \input, \externaldocument
%
% -- FIGURES AND TABLES
%
% Include table/figure captions directly after paragraph where first cited.
%
% NO GRAPHICS
% - Figures should be uploaded separately from your manuscript file. 
% - Extract figures (in PDF) generated by LaTeX before submission.
% - Combine multiple panels/subfigures into one image before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column. 
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2". 
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % % 
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8]{inputenc}
%\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Leave date blank
\date{}

% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
\setlength{\headheight}{27.023pt}
\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
%\rfoot{\thepage/\pageref{lastpage}}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\sf PLOS}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION


\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Mutation Rate as a Selection Mechanism in Continuous Evolution of Bacteriophage} % Title Case
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Peter Reintjes\textsuperscript{1}
\\
\bigskip
\textbf{1} Museum of Life and Science, Durham, North Carolina, USA
\bigskip

% Additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
% 
% Remove or comment out the author notes below if they aren't used.
%
% Primary Equal Contribution Note
%\Yinyang These authors contributed equally to this work.

% Additional Equal Contribution Note
% Also use this double-dagger symbol for special authorship notes, such as senior authorship.
%\ddag These authors also contributed equally to this work.

% Current address notes
%\textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% \textcurrency b Insert second current address 
% \textcurrency c Insert third current address

% Deceased author note
%\dag Deceased

% Group/Consortium Author Note
%\textpilcrow Membership list can be found in the Acknowledgments section.

% Use the asterisk to denote corresponding authorship and provide email address in note below.
%* correspondingauthor@institute.edu

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Engineering novel proteins via continuous evolution of bacteriophage currently requires the host bacteria to be transformed with two additional functions: A mutagenesis vector to provide an elevated rate of viral mutation, and a selection mechanism to increase the number of infectious progeny for the mutant encoding the improved product\cite{pace}.  I propose using a selection mechanism which simply modulates the rate of mutation by producing a mutagenesis suppressor from the desired activity of the evolving protein.  Improvements in the desired activity will result in a higher percentage of faithful copies of the parent genome in the succeeding generation.

As the evolving protein approaches a high level of desired activity, the proportionally lowered mutation rate will ensure that more viral progeny contain exact copies of the parent genotype while a baseline mutation rate ensures that our mechanism to lower the mutation rate will not result in stagnation.

Extremely high in vivo mutation rates are now possible\cite{mutation}.  With M-selection, these high rates will correspond to little or none of the desired activity, these lines of descent will be subject to catastrophic levels of mutation and low probabilities of faithful reproduction. The modulation of mutation rate within each host cell means that one of these wildly mutating strains could find an environment for faithful reproduction if it should stumble upon a promising mutation in the evolving gene.


% Please keep the Author Summary between 150 and 200 words
% Use first person. PLOS ONE authors please skip this step. 
% Author Summary not valid for PLOS ONE submissions.   
%\section*{Author Summary}
%Lorem ipsum dolor sit amet,

\linenumbers

% Use "Eq" instead of "Equation" for equation citations.
\section*{Introduction}
\chapter{M-Selection}
\section{Introduction}
Continuous evolution of bacteriophage has become a potent protein engineering
tool.  For example, PACE - Phage Assisted Continuous Evolution described by
Esvelt,Carlson and Lui. 

The continuous evolution process produces a viral genome containing a gene
which has undergone many generations of mutation and selective pressure for
a particular property.  The generality of this approach to protein engineering
is only limited by our ability to create a selection mechanism for the desired
property and insert an expressable initial form of the gene into the virus.

Specifically, the PACE system requires:
\begin{itemize}

\item{}
A modified viral genome missing a gene required for infectious progeny.

\item{}
A modified host genome/plasmid to induce an elevated mutation rate.

\item{}
A modified host genome/plasmid to provide a selection mechanism to provide
the missing gene product in proportion to the desired activity of the evolving
gene.
\end{itemize}

Effectively, the number of infectious progeny is proportional to the activity
of the selection mechanism. In principle, if the current mutation of the gene
produces a product with none of the desired activity, none of the infectivity
protein will be produced by the host, and there will be no infectous progeny.
This process involves a continuous flow of host cells through a reaction vessel
and so any genotype producing less than two infectious virions per host will
be washed out.

M-Selection is proposed to the selection mechanism by combining it with
the mechanism for elevated mutation.

The benefits of M-Selection may include:

The virus is not crippled, but only has an additional gene or domain, similar to
phage display.

The nominal (high) mutation rate can be chosen to guarantee that there will be
a minimum number of non-mutant copies produced for every infection.  This percentage of clones will then increase as the mutation rate is reduced.



For example, if the selection mechanism stimulated by the desired activity will consist
of a repressor for the production of mutagenic polymerase II-.
Positive selection is the result of reduced mutation, resulting in more clones of the
selected virus relative to its mutants.

As far as we know, modulation of the mutation rate within a particular host has
not been exploited in this way.  This form of albeit un-natural selection is referred to
as M-selection, whereby the selection mechanism consists of reducing the rate of mutation
acting on a particular instance of the viral genome precisely because it is having some
desired behavior.

Figure 1 shows how the combined mutagenic and selection plasmid takes the place of the AP and MP in PACE.

\def\boxone#1{
  \put(19,30){\line(1,0){67}}     % horiz top
  \put(19,0){\line(1,0){67}}      % horiz bottom
  \put(20,0){\line(0,1){30}}      % vert left
  \put(85,0){\line(0,1){30}}      % vert right
  \put(25,15){#1}
}
\def\boxtwo#1#2{
  \put(19,30){\line(1,0){80}}     % horiz top
  \put(19,0){\line(1,0){80}}      % horiz bottom
  \put(20,0){\line(0,1){30}}      % vert left
  \put(98,0){\line(0,1){30}}      % vert right
  \put(25,17){#1}
  \put(25,7){#2}
}
\def\circleone#1#2{\circle{#1}\put(-17,-4){#2}}
\def\circletwo#1#2#3{\circle{#1}\put(-17,0){#2}\put(-15,20){#3}}

\picture(200,150)(-70,0)

\put(90,90){\vector(0,-1){15}}
\put(90,30){\vector(0,1){15}}

\put(20,60){\vector(1,0){30}}
\put(130,60){\vector(1,0){30}}
\put(130,60){\vector(1,1){30}}
\put(130,60){\vector(1,-1){30}}
\linethickness{2pt}
\thicklines
\put(0,60){\circleone{40}{\Large{\ \ P1}}}
\put(30,90){\boxtwo{\ Phage Shock}{\ \ \ Promoter}}
\put(30,45){\boxtwo{MUTAGENESIS}{\ \ Replication}}
\put(30,0){\boxtwo{Antisense RNA}{\ \ Suppressor}}

\put(180,60){\circleone{40}{\Large{\ \ F1}}}

\endpicture
\vskip 1.0cm
\centerline{\bf Fig 1. M-selection Plasmid}
\vskip 0.2cm

\subsection{Difficulties of working with crippled Virus genome}

\begin{eqnarray}
\label{eq:schemeP}
	\mathrm{P_Y} = \underbrace{H(Y_n) - H(Y_n|\mathbf{V}^{Y}_{n})}_{S_Y} + \underbrace{H(Y_n|\mathbf{V}^{Y}_{n})- H(Y_n|\mathbf{V}^{X,Y}_{n})}_{T_{X\rightarrow Y}},
\end{eqnarray}

\subsection*{Multiplicity of Infection}

\vskip 0.2cm
\begin{eqnarray}
\label{eq:MOI}
\mathrm{P}\left( x \right) = \frac{{e^{ - \lambda } \lambda ^x }}{{x!}}
\end{eqnarray}

\section{Comparison with Natural Selection}
Selection pressure for an optimal mutation rate has its own natural history\cite{evomut} although for settled 
A selection mechanism for lowered mutation rate
Natural selection is largely about responses to the external environment, which we control closely in laboratory evolution.
The closest parallel would be found in species which can reproduce by asexual or sexual reproduction and change the rates between the alternatives as a way of accessing more diversity through sexual reproduction or producing more clones via asexual reproduction.


\section*{Materials and Methods}
\subsection*{Components of Statistical Analysis}
Percentage of activated genes due to Phage Shock promoter.
Number of polII-  (mutagenic polymerase) transcripts * error rate due to mutagenic polymerase.

Error rate of normal polII
Error rate of polII- 

Number of phage particles produced np(0-6 min) = 0 (six minutes before any phage appear),
then production between 75-200 per hour.

Initial single-strand (virion) DNA production occurs in non-mutagenic environment, then mutants appear along with polII- production, until suppression of polII- occurs due to binding.  Binding ? anti-sense RNA transcription ? polII-  repression ? reduction in mutations.


% For figure citations, please use "Fig" instead of "Figure".
Nulla mi mi, Fig~\ref{fig1} venenatis sed ipsum varius.

% Place figure captions after the first paragraph in which they are cited.
\begin{figure}[!h]
\caption{{\bf Bold the figure title.}
Figure caption text here, please use this space for the figure panel descriptions instead of using subfigure commands. A: Lorem ipsum dolor sit amet. B: Consectetur adipiscing elit.}
\label{fig1}
\end{figure}

% Results and Discussion can be combined.
\section*{Results}
Nulla mi mi.

% Place tables after the first paragraph in which they are cited.
\begin{table}[!ht]
\begin{adjustwidth}{-2.25in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
\centering
\caption{
{\bf Table caption Nulla mi mi, venenatis sed ipsum varius, volutpat euismod diam.}}
\begin{tabular}{|l+l|l|l|l|l|l|l|}
\hline
\multicolumn{4}{|l|}{\bf Heading1} & \multicolumn{4}{|l|}{\bf Heading2}\\ \thickhline
$cell1 row1$ & cell2 row 1 & cell3 row 1 & cell4 row 1 & cell5 row 1 & cell6 row 1 & cell7 row 1 & cell8 row 1\\ \hline
$cell1 row2$ & cell2 row 2 & cell3 row 2 & cell4 row 2 & cell5 row 2 & cell6 row 2 & cell7 row 2 & cell8 row 2\\ \hline
$cell1 row3$ & cell2 row 3 & cell3 row 3 & cell4 row 3 & cell5 row 3 & cell6 row 3 & cell7 row 3 & cell8 row 3\\ \hline
\end{tabular}
\begin{flushleft} Table notes Phasellus venenatis, tortor nec vestibulum mattis, massa tortor interdum felis, nec pellentesque metus tortor nec nisl. Ut ornare mauris tellus, vel dapibus arcu suscipit sed.
\end{flushleft}
\label{table1}
\end{adjustwidth}
\end{table}

\subsection*{\lorem\ and \ipsum\ Nunc blandit a tortor.}
\subsubsection*{3rd Level Heading.} 
Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque. Quisque augue sem, tincidunt sit amet feugiat eget, ullamcorper sed velit. Sed non aliquet felis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris commodo justo ac dui pretium imperdiet. Sed suscipit iaculis mi at feugiat. 

\begin{enumerate}
	\item{react}
	\item{diffuse free particles}
	\item{increment time by dt and go to 1}
\end{enumerate}


\subsection*{Sed ac quam id nisi malesuada congue.}

Nulla mi mi, venenatis sed ipsum varius, volutpat euismod diam. Proin rutrum vel massa non gravida. Quisque tempor sem et dignissim rutrum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at justo vitae nulla elementum commodo eu id massa. In vitae diam ac augue semper tincidunt eu ut eros. Fusce fringilla erat porttitor lectus cursus, vel sagittis arcu lobortis. Aliquam in enim semper, aliquam massa id, cursus neque. Praesent faucibus semper libero.

\begin{itemize}
	\item First bulleted item.
	\item Second bulleted item.
	\item Third bulleted item.
\end{itemize}

\section*{Discussion}
Nulla mi mi, venenatis sed ipsum varius, Table~\ref{table1} volutpat euismod diam. Proin rutrum vel massa non gravida. Quisque tempor sem et dignissim rutrum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at justo vitae nulla elementum commodo eu id massa. In vitae diam ac augue semper tincidunt eu ut eros. Fusce fringilla erat porttitor lectus cursus, vel sagittis arcu lobortis. Aliquam in enim semper, aliquam massa id, cursus neque. Praesent faucibus semper libero~\cite{bib3}.

\section*{Conclusion}

I think this will work.

\section*{Supporting Information}

% Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
\paragraph*{S1 Fig.}
\label{S1_Fig}
{\bf Bold the title sentence.} Add descriptive text after the title of the item (optional).

\paragraph*{S2 Fig.}
\label{S2_Fig}
{\bf Lorem Ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 File.}
\label{S1_File}
{\bf Lorem Ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 Video.}
\label{S1_Video}
{\bf Lorem Ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 Appendix.}
\label{S1_Appendix}
{\bf Lorem Ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 Table.}
\label{S1_Table}
{\bf Lorem Ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\section*{Acknowledgments}
Cras egestas velit mauris, eu mollis turpis pellentesque sit amet. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam id pretium nisi. Sed ac quam id nisi malesuada congue. Sed interdum aliquet augue, at pellentesque quam rhoncus vitae.

\nolinenumbers

% Either type in your references using
% \begin{thebibliography}{}
% \bibitem{}
% Text
% \end{thebibliography}
%
% or
%
% Compile your BiBTeX database using our plos2015.bst
% style file and paste the contents of your .bbl file
% here.
% 
\begin{thebibliography}{10}

\bibitem{pace}
Esvelt K. M., Carlson J. C. \& Liu D. R.
\newblock {A system for the continuous directed evolution of biomolecules}.
\newblock Nature 472, 499–503 (2011) .

\bibitem{mutation}
Badran AH, Liu DR.
\newblock {{D}evelopment of potent in vivo mutagenesis plasmids with broad mutational spectra}.
\newblock Nature Communications. 2015;6:8425. doi:10.1038/ncomms9425.

\bibitem{evomut}
Lynch M.
\newblock {{E}volution of the Mutation Rate}.
\newblock Trends in Genetics, 2010 August ; 26(8): 345–352. doi:10.1016/j.tig.2010.05.003.
\end{thebibliography}



\end{document}



